scripts/run_docker

```diff
@@ -48,13 +48,30 @@ if [ "$OSTYPE" == "msys" ]; then
   CONTAINER_RUNTIME="winpty docker"
 fi
 
-RAND_NAME=$(env LC_ALL=C tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 16 | head -n 1)
-if [ -z "$NETWORK_NAME" ]; then
-  echo "No Docker network specified."
-  $CONTAINER_RUNTIME run --rm -ti --name "aries-cloudagent-runner_${RAND_NAME}" \
-    $ARGS aries-cloudagent-run "$@"
+ACAPY_CONTAINER_NAME="${CONTAINER_NAME}"
```

Comment:
I find this assignment and then the following mixed use of the two variables confusing.

Reply:
I was just following previous code but you are right, removed.